ansible/roles/ironic/tasks/config.yml

```diff
@@ -90,8 +90,8 @@
   when:
     # Only required when Ironic inspector is in use.
     - groups['ironic-inspector'] | length > 0
-    - inventory_hostname in groups['ironic-pxe']
-    - not enable_ironic_ipxe | bool
+    - inventory_hostname in groups['ironic-pxe'] or
+        inventory_hostname in groups['ironic-ipxe']
```

Comment:
I think this was correct before - kernel and ramdisk should be loaded via iPXE only when enabled.